controllers/front/notify.php

```diff
@@ -113,18 +130,24 @@ public function postProcess()
     }
 
     /**
-     * @param int $cartId
-     *
-     * @return AssertBody
-     * @throws Exception
+     * @param $cartId
+     * @return void
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
      */
-    private function assertTransaction($cartId)
+    private function authorizeSaferPayOrderEntity($cartId)
```

Comment:
Not sure if deleting all the assertTransaction function was the right choice :D

Reply:
Additionally, this seems like an Action class and should not exist in controllers.